lib/pdk/module/templatedir.rb

```diff
@@ -236,26 +232,44 @@ def self.files_in_template(dirs)
       #
       # @api private
       def config_for(dest_path)
```

Comment:
It would be cool if the path to `sync.yml` could be passed in as an optional parameter here, and if it is nil, then the code below can default the path to the `<dest_path>/sync.yml`. This will allow us to potentially add support for externally provided `sync.yml` config in the future.